lib/lightning_web/live/job_live/job_builder.ex

```diff
@@ -208,12 +221,55 @@ defmodule LightningWeb.JobLive.JobBuilder do
           >
             Save
           </Form.submit_button>
+          <%= if @job_id != "new" do %>
+            <Common.button
+              id="delete-job"
+              text="Delete"
+              phx-click="delete"
+              phx-target={@myself}
+              phx-value-id={@job_id}
+              disabled={!is_deletable(@job_id)}
+              data={[
+                confirm:
+                  "This action is irreversible, are you sure you want to continue?"
+              ]}
+              title={delete_job_title(@job_id)}
```

Comment:
Instead of calling a function here, use an expression like `if @is_deleteable, do: "x", else: "y"`